viewer/src/__tests__/culling/ByVisibilityGpuSectorCuller.test.ts

```diff
@@ -2,18 +2,21 @@
  * Copyright 2020 Cognite AS
  */
 import * as THREE from 'three';
-import { OrderSectorsByVisibilityCoverage } from '../../dataModels/cad/internal/sector/culling/OrderSectorsByVisibilityCoverage';
-import { ByVisibilityGpuSectorCuller } from '../../dataModels/cad/internal/sector/culling/ByVisibilityGpuSectorCuller';
-import { CadModel } from '../../dataModels/cad/internal';
-import { SectorMetadata } from '../../dataModels/cad/internal/sector/types';
-import { ModelDataRetriever } from '../../utilities/networking/ModelDataRetriever';
-import { SectorSceneImpl } from '../../dataModels/cad/internal/sector/SectorScene';
 import { mat4, vec3 } from 'gl-matrix';
+
+import { ModelDataRetriever } from '@/utilities/networking/ModelDataRetriever';
```

Comment:
Would it be possible to do the following for alias'?
@dataModels would be linked to ['src/dataModels/*]
@utilities ... ['src/utilities/*']
...

or go even further and just have @cad, @pointcloud etc.

Sure, it would make us have to add a new alias for packages, but just @ in the beginning of the import just seems weird. @root maybe, or @src?

Reply:
I believe using `@/...` is a convention that is quite common.

As for `@cad` etc I think we should introduce sub-packages to achieve something like this, not a ton of aliases.